packages/@glimmerx/babel-plugin-component-templates/index.js

```diff
@@ -91,8 +121,8 @@ module.exports = function(babel, options) {
           bindings.map(binding => {
             const { identifier } = binding;
             binding.reference(path);
-            return t.objectProperty(identifier, identifier, false, true);
-          })
+            return t.objectProperty(t.identifier(identifier.name), identifier, false, false);
```

Comment:
prev notation won't reference to original import (at least, in parcel)

```
import MyComponent from './foo';
scope: () => ({ MyComponent })
```
transformed to 

```
import _MyComponent from './foo';
scope: () => ({ MyComponent })
```
and it's never binded

new transform binding works as expected:

```
import _MyComponent from './foo';
scope: () => ({ MyComponent: _MyComponent })
```